autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
 
-def parse_section(section):
-    " parse_course() must be done first, and pass the data for each section here "
+def parse_section(course):
+    #gets values to create object
+    section_id = ''
+    section_subject = course['subject']
+    course_number = course['courseNumber']
+    section_number = ''
+    section_term_code = course['term']
+    
+    section_min_credits = course['creditHourLow']
+    section_max_credits = course['creditHourHigh']
 
-def parse_meeting(meeting):
-    " parse_section() must be done first, and pass the data for each meeting here"
+    section_max_enrollment = course['maximumEnrollment']
+    section_current_enrollment = course['enrollment']
```

Comment:
Are we actually going to use the current and max enrollment for anything? I don't see how this is really going to help create schedule unless we're doing prioritize class size or something.

Reply:
We won't want to create schedules with sections that are full in them, so enrollment will definitely be useful. We might want to display that and the max enrollment to the user too, but if we're not I agree that I don't see a point in storing it.